docs/fit/index.rst

```diff
@@ -765,22 +832,25 @@ of parameters by setting the ``parlist`` parameter of the
 :py:meth:`~sherpa.fit.Fit.est_errors` call. As an example, the
 errors on just :math:`c_1` can be evaluated with the following:
 
-   >>> c1errs = f.est_errors(parlist=(mdl.c1, ))
-   polynom1d.c1 lower bound:	-0.250889
-   polynom1d.c1 upper bound:	0.250889
-   >>> print(c1errs)
-   datasets    = None
-   methodname  = confidence
-   iterfitname = none
-   fitname     = levmar
-   statname    = chi2gehrels
-   sigma       = 1
-   percent     = 68.26894921370858
-   parnames    = ('polynom1d.c1',)
-   parvals     = (-2.4169154937357664,)
-   parmins     = (-0.25088931712955054,)
-   parmaxes    = (0.25088931712955054,)
-   nfits       = 2
+.. plot::
```

Comment:
My assumption is that you don't actually need this `plot` directive (since it should not be relevant for the next plot, as `IntervalProjection` should not be affected by this), but I can not guarantee it. And I'm too lazy to check myself.